lib/constants.js

```diff
@@ -6,6 +6,7 @@ exports.VERSION = pkg.version;
 
 exports.DEFAULT_PORT = process.env.PORT || 9876;
 exports.DEFAULT_HOSTNAME = process.env.IP || 'localhost';
+exports.DEFAULT_LISTEN_IP = process.env.IP || '0.0.0.0';
```

Comment:
I don't really like the magical env variable as discussed in #1186, should we just leave `0.0.0.0` as default value and not read the `IP` env variable?